.github/workflows/ci.yml

```
@@ -36,6 +36,11 @@ jobs:
       # - name: Add msbuild to PATH
       #   if: ${{ matrix.os == 'windows-latest' }}
       #   uses: microsoft/setup-msbuild@v1.0.2
+      - name: Verify CMake build
+        run: |
+          mkdir build && cd build
+          FC=${{ matrix.compiler }} cmake ../
+          make
```

```suggestion
      - name: Verify CMake build
        run: |
          mkdir build && cd build
          cmake ../
          make
        env:
          FC: ${{ matrix.compiler }}
```
The current build fails on Windows due to the different way how environment variables are set there (see [error log](https://github.com/trixi-framework/FTObjectLibrary/pull/14/checks?check_run_id=2877784509#step:3:12)). Let's see if this fixes the issues.